tst/com/amazon/corretto/crypto/provider/test/AesTest.java

```diff
@@ -579,6 +581,24 @@ public void test_initParameters() throws Throwable {
         assertArrayEquals(PLAINTEXT, decrypted);
     }
 
+    @SuppressWarnings("ConstantConditions")
+    @Test
+    public void test_initNullKey() throws Throwable {
+        jceC.init(Cipher.ENCRYPT_MODE, key);
```

Comment:
Since this test will fail on earlier versions of ACCP, we need to add a check at the beginning of the test which bails out early (without failing) if an older version is under test.

```suggestion
        assumeMinimumVersion("1.6.0", AmazonCorrettoCryptoProvider.INSTANCE);
        jceC.init(Cipher.ENCRYPT_MODE, key);
```